EIPS/eip-1052.md

```diff
@@ -18,16 +18,48 @@ Many contracts need to perform checks on a contract's bytecode, but do not neces
 Contracts can presently do this using the `EXTCODECOPY` opcode, but this is expensive, especially for large contracts, in cases where only the hash is required. As a result, we propose a new opcode, `EXTCODEHASH`, which returns the keccak256 hash of a contract's bytecode.
 
 ## Specification
-A new opcode, `EXTCODEHASH`, is introduced, with number 0x3D. `EXTCODEHASH` takes one argument from the stack, and pushes the keccak256 hash of the code at that address to the stack.
+
+A new opcode, `EXTCODEHASH`, is introduced, with number 0x3D. The `EXTCODEHASH` 
+takes one argument from the stack, discards the first 96 bits 
```

Comment:
Probably "zeroes the most significant 96 bits" is clearer.

Reply:
Changed.